### PR TITLE
fix(timetracker): refactoring of the timetracker documentation

### PR DESCRIPTION
**Pull request by danila-m**
Closes [BS-16544](https://bonitasoft.atlassian.net/browse/BS-16544)

### CODE REVIEW

File: md/performance-tuning.md [hunk 1]
```diff
@@ -406,13 +406,39 @@ Be sure to configure these appropriately.
...
+#bonita.tenant.timetracker.memory.maxSize=1000000
+```
+To activate connector time tracking:
+1. Uncomment all the previous lines except ```## Time tracker```.
+2. Change the value of `startTracking` from `false` to `true`.
+
+The other parameters can be left at their default value, left commented, or set to the desired value. What each of them does:
+
+1. `maxSize` maximum of records that will be saved by the time tracker before a flush.
+2. `flushIntervalInSeconds` the interval beetween two flushes on the timetracker thread.
```
Comment (nchabanoles) [inline md/performance-tuning.md]: If flushIntervalInSeconds and maxSize are defined what happens? Is there a flush every flushIntervalInSecondsand AND everytime maxSize is reached before the interval complete?

Or those attributes should not be defined at the same time?

File: md/performance-tuning.md [hunk 2]
```diff
@@ -406,13 +406,39 @@ Be sure to configure these appropriately.
...
+3. `csv.activateAtStart` wether to save the result of the timetracker into a csv file.
+4. `csv.folder` the folder where to save the csv file.
+5. `csv.separator` the separator character in the csv file
+6. `memory.activateAtStart` wether to save the result of the timetracker in memory.
+7. `memory.maxSize` maximum amount of records saved in memory
```
Comment (nchabanoles) [inline md/performance-tuning.md]: What happens when the maxSize is reached? Older record are lost? Is there a flush on CSV?
Can we have csv.activeAtStart and memory.activeAtStart enable at the same time?

**Review by nchabanoles — APPROVED** [review]
Only few questions but current changes are already bringing more information than previsous version.

**Comment by danila-m** [comment]
"Can we have csv.activeAtStart and memory.activeAtStart enable at the same time?" the answer is yes. I added more precision in the description of the maxsizes options.

**Comment by nchabanoles** [comment]
LGTM

**Pull request merged**